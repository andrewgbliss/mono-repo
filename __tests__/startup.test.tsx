import { render } from '@testing-library/react';
import Startup from '../pages/startup';

it('renders Startup unchanged', () => {
  const { container } = render(<Startup />);
  expect(container).toMatchSnapshot();
});
