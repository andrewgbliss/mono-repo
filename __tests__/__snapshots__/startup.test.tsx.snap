// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders Startup unchanged 1`] = `
<div>
  <header>
    <div
      class="absolute w-full fade-in"
    >
      <section>
        <div
          class="flex justify-between items-center w-full sm:py-8"
        >
          <img
            class="w-20"
            src="img/startup/logo.png"
          />
          <nav
            class="text-xl text-white"
          >
            <a
              class="hover:underline"
              href="#"
            >
              Get Early Access
            </a>
          </nav>
        </div>
      </section>
    </div>
  </header>
  <div
    class="bg-banner-bk text-white text-center md:text-left"
  >
    <section
      class="fade-in"
    >
      <div
        class="h-32"
      />
      <h1
        class="pady"
      >
        A New Way
        <br />
        To Start Business
      </h1>
      <p
        class="pady"
      >
        Lorem ipsum, dolor sit amet consectetur adipisicing elit. Tempora dolor adipisci
        <br />
        dicta dolores qui quaerat iure cumque perspiciatis mollitia accusamus ex aliquam et.
      </p>
      <div
        class="pady"
      >
        <button
          class="btn-primary"
        >
          Get Started
        </button>
      </div>
      <div
        class="h-8 sm:h-16"
      />
    </section>
  </div>
  <div
    class="text-center"
  >
    <section>
      <div
        class="py-16"
      >
        <h2>
          Smartest protection for your site
        </h2>
      </div>
      <div
        class="grid grid-cols-1 md:grid-cols-3"
      >
        <div
          data-aos="fade-up"
          data-aos-delay="0"
          data-aos-duration="1000"
          data-aos-once="true"
        >
          <img
            class="mx-auto"
            src="img/startup/smart-protect-1.jpg"
          />
          <h4>
            Anti-Spam
          </h4>
          <div
            class="pad"
          >
            <p
              class="text-lg"
            >
              Lorem ipsum dolor sit amet porro his no his deleniti
            </p>
          </div>
        </div>
        <div
          data-aos="fade-up"
          data-aos-delay="200"
          data-aos-duration="1000"
          data-aos-once="true"
        >
          <img
            class="mx-auto"
            src="img/startup/smart-protect-2.jpg"
          />
          <h4>
            Phishing Detect
          </h4>
          <div
            class="pad"
          >
            <p
              class="text-lg"
            >
              Ne error antiopam usu. Sed vocen concludaturque ea
            </p>
          </div>
        </div>
        <div
          data-aos="fade-up"
          data-aos-delay="400"
          data-aos-duration="1000"
          data-aos-once="true"
        >
          <img
            class="mx-auto"
            src="img/startup/smart-protect-3.jpg"
          />
          <h4>
            Smart Scan
          </h4>
          <div
            class="pad"
          >
            <p
              class="text-lg"
            >
               
              Et usu ocurreret elaboraret doctus prodesse assueverit.
            </p>
          </div>
        </div>
      </div>
    </section>
  </div>
  <div
    class="bg-gradient text-center md:text-left"
  >
    <section>
      <div
        class="py-12 lg:py-24"
      >
        <div
          class="flex flex-wrap-reverse"
        >
          <div
            class="w-full md:w-1/2"
          >
            <div
              data-aos="fade-right"
              data-aos-delay="0"
              data-aos-duration="1000"
              data-aos-once="true"
            >
              <h2
                class="pt-6 md:pt-0"
              >
                Take a look inside
              </h2>
              <div
                class="pady"
              >
                <p>
                  Te iisque labitur eos, nec sale argumentum scribentur no, augue disputando in vim. Erat fugit sit at, ius lorem deserunt deterruisset no.
                </p>
              </div>
              <div
                class="pady-thin"
              >
                <button
                  class="btn-secondary"
                >
                  Learn More
                </button>
              </div>
            </div>
          </div>
          <div
            class="w-full md:w-1/2"
          >
            <div
              data-aos="fade-left"
              data-aos-delay="0"
              data-aos-duration="1000"
              data-aos-once="true"
            >
              <img
                class="mx-auto"
                src="/img/startup/feature-1.png"
              />
            </div>
          </div>
        </div>
      </div>
    </section>
  </div>
  <div
    class="bg-primary text-center md:text-left"
  >
    <section>
      <div
        class="py-8 lg:py-16"
      >
        <div
          class="flex flex-wrap"
        >
          <div
            class="w-full md:w-1/2"
          >
            <div
              data-aos="fade-right"
              data-aos-delay="500"
              data-aos-duration="1000"
              data-aos-once="true"
            >
              <img
                class="mx-auto sm:mx-0"
                src="/img/startup/feature-2.png"
              />
            </div>
          </div>
          <div
            class="w-full md:w-1/2"
          >
            <div
              data-aos="fade-left"
              data-aos-delay="500"
              data-aos-duration="1000"
              data-aos-once="true"
            >
              <h2
                class="pt-6 md:pt-0"
              >
                Safe and reliable
              </h2>
              <div
                class="pady"
              >
                <p>
                  Duo suas detracto maiestatis ad, commodo lucilius invenire nec ad, eum et oratio disputationi. Falli lobortis his ad
                </p>
              </div>
              <div
                class="pady-thin"
              >
                <button
                  class="btn-secondary"
                >
                  Learn More
                </button>
              </div>
            </div>
          </div>
        </div>
      </div>
    </section>
  </div>
  <div
    class="text-center"
  >
    <section
      class="p-0"
    >
      <div
        class="p-0 sm:p-6 pb-16 sm:pb-16"
      >
        <div
          class="pt-16 sm:py-16 px-6"
        >
          <h2>
            Check our Pricing
          </h2>
        </div>
        <div
          class="grid grid-cols-1 md:grid-cols-3 text-center"
        >
          <div
            data-aos="fade-right"
            data-aos-delay="0"
            data-aos-duration="1000"
            data-aos-once="true"
          >
            <div
              class="py-16"
            >
              <h5>
                STARTUP
              </h5>
              <div
                class="pt-4"
              >
                <span
                  class="font-bold"
                >
                  $
                </span>
                <span
                  class="text-6xl font-bold"
                >
                  0
                </span>
                <span
                  class="font-bold"
                >
                  / MO.
                </span>
              </div>
              <div
                class="pad"
              >
                <p>
                  Up to 5 Documents
                </p>
                <p>
                  Up to 3 Reviews
                </p>
                <p>
                  5 Team Members
                </p>
                <p>
                  Limited Support
                </p>
              </div>
              <div
                class="pad"
              >
                <button
                  class="btn"
                >
                  Get Free
                </button>
              </div>
            </div>
          </div>
          <div
            class="bg-secondary text-white"
            data-aos="fade-up"
            data-aos-delay="0"
            data-aos-duration="1000"
            data-aos-once="true"
          >
            <div
              class="py-16"
            >
              <h5>
                PREMIUM
              </h5>
              <div
                class="pt-4"
              >
                <span
                  class="font-bold"
                >
                  $
                </span>
                <span
                  class="text-6xl font-bold"
                >
                  10
                </span>
                <span
                  class="font-bold"
                >
                  / MO.
                </span>
              </div>
              <div
                class="pad"
              >
                <p>
                  Up to 15 Documents
                </p>
                <p>
                  Up to 10 Reviews
                </p>
                <p>
                  25 Team Members
                </p>
                <p>
                  Limited Support
                </p>
              </div>
              <div
                class="pad"
              >
                <button
                  class="btn-primary"
                >
                  Get Free
                </button>
              </div>
            </div>
          </div>
          <div
            data-aos="fade-left"
            data-aos-delay="0"
            data-aos-duration="1000"
            data-aos-once="true"
          >
            <div
              class="py-16"
            >
              <h5>
                PROFESSIONAL
              </h5>
              <div
                class="pt-4"
              >
                <span
                  class="font-bold"
                >
                  $
                </span>
                <span
                  class="text-6xl font-bold"
                >
                  30
                </span>
                <span
                  class="font-bold"
                >
                  / MO.
                </span>
              </div>
              <div
                class="pad"
              >
                <p>
                  Unlimited Documents
                </p>
                <p>
                  Unlimited Reviews
                </p>
                <p>
                  Unlimited Members
                </p>
                <p>
                  Unlimited Support
                </p>
              </div>
              <div
                class="pad"
              >
                <button
                  class="btn"
                >
                  Get Free
                </button>
              </div>
            </div>
          </div>
        </div>
      </div>
    </section>
  </div>
  <div
    class="text-center bg-gray-200"
  >
    <section>
      <div
        class="grid grid-cols-1 sm:grid-cols-2 md:grid-cols-3 lg:grid-cols-6 gap-4"
      >
        <div
          class="pad flex justify-center items-center"
        >
          <img
            src="/img/startup/client-1.png"
          />
        </div>
        <div
          class="pad flex justify-center items-center"
        >
          <img
            src="/img/startup/client-2.png"
          />
        </div>
        <div
          class="pad flex justify-center items-center"
        >
          <img
            src="/img/startup/client-3.png"
          />
        </div>
        <div
          class="pad flex justify-center items-center"
        >
          <img
            src="/img/startup/client-4.png"
          />
        </div>
        <div
          class="pad flex justify-center items-center"
        >
          <img
            src="/img/startup/client-5.png"
          />
        </div>
        <div
          class="pad flex justify-center items-center"
        >
          <img
            src="/img/startup/client-6.png"
          />
        </div>
      </div>
    </section>
  </div>
  <div
    class="bg-contact-bk text-white text-center sm:text-left"
  >
    <section>
      <div
        class="py-16"
      >
        <div
          class="grid grid-cols-1 md:grid-cols-2"
        >
          <div>
            <h2
              class="pb-6"
            >
              Contact Us
            </h2>
            <p
              class="pady"
            >
              Te iisque labitur eos, nec sale argumentum scribentur, augue disputando in vim. Erat fugit sit at, ius lorem.
            </p>
            <p
              class="pady-thin"
            >
              Email : company@email.com
              <br />
              Phone : 489-613-4200
              <br />
              Address : 987 Black Avenue, Lehi, Utah
            </p>
          </div>
          <div
            class="pt-6 sm:pt-0"
          >
            <div
              class="grid grid-cols-2 gap-4 text-primary"
            >
              <div>
                <label
                  class="block text-sm font-bold mb-2"
                >
                  Your Name
                </label>
                <input
                  name="name"
                />
              </div>
              <div>
                <label
                  class="block text-sm font-bold mb-2"
                >
                  Your Email
                </label>
                <input
                  name="email"
                />
              </div>
              <div
                class="col-span-2"
              >
                <label
                  class="block text-sm font-bold mb-2"
                >
                  Message
                </label>
                <textarea
                  name="email"
                />
              </div>
              <div
                class="col-span-2"
              >
                <button
                  class="btn-primary w-full"
                >
                  Submit
                </button>
              </div>
            </div>
          </div>
        </div>
      </div>
    </section>
  </div>
  <footer
    class="text-white"
  >
    <div
      class="bg-secondary"
    >
      <section>
        <div
          class="grid grid-cols-1 md:grid-cols-2"
        >
          <div
            class="flex justify-center md:justify-start items-center"
          >
            Copyright © 
            2022
             Entry Level Developer Training
          </div>
          <div
            class="flex justify-center md:justify-end items-center pt-6 md:pt-0"
          >
            <div
              class="pad-thin"
            >
              <a
                class="btn-social-media"
                href="https://www.youtube.com/channel/UCHo5peSzuo7c19MB69QJYdA"
                rel="noopener noreferrer"
                target="_blank"
              >
                <i
                  class="fab fa-youtube-square"
                />
              </a>
            </div>
            <div
              class="pad-thin"
            >
              <a
                class="btn-social-media"
                href="https://www.instagram.com/entryleveldevelopertraining"
                rel="noopener noreferrer"
                target="_blank"
              >
                <i
                  class="fab fa-instagram"
                />
              </a>
            </div>
            <div
              class="pad-thin"
            >
              <a
                class="btn-social-media"
                href="https://twitter.com/entryleveldeve1"
                rel="noopener noreferrer"
                target="_blank"
              >
                <i
                  class="fab fa-twitter"
                />
              </a>
            </div>
          </div>
        </div>
      </section>
    </div>
  </footer>
</div>
`;
